daprdocs/content/en/operations/extending/developing-grpc-components.md

```diff
@@ -6,8 +6,45 @@ weight: 250
 description: "Extending Dapr with external gRPC-based components"
 ---
 
-[gRPC-based](https://grpc.io/) Dapr components are typically run as containers or processes that communicate with the Dapr main process via [Unix Domain Sockets](https://en.wikipedia.org/wiki/Unix_domain_socket).
+## Components and Pluggable Components
 
+Out of the box, Dapr [building blocks] come with integrations to a wide range of cloud providers and open source solutions. We call each  of these individual integrations _"components"_. For instance, a input biding integration for MySql is a _component_ and a state store integration for MySQL would be a different component. 
```

Comment:
How about link components to the concepts page? https://docs.dapr.io/concepts/components-concept/